### DISCUSSION TITLE
Classic ASP / VBScript Support in Copilot

### DISCUSSION THREAD

**Discussion by WakingDragon** [community/community#53154] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Does Copilot have anything to offer if working on a Classic ASP website? ChatGPT has a fairly good stab at Ye Olde Language, but I would like to know if Copilot is worth looking into for this.

Ta,
Ben

**Comment by Akash1134** [comment, marked as answer]
Hi @WakingDragon, 

GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks, but works especially well for Python, JavaScript, TypeScript, Ruby, Go, C# and C++. 

Copilot supports Classic ASP and VB Script. Note that the suggestion quality may vary by language and depending on the amount of content (i.e. surrounding code) available in the code editor.
